frontend/app/services/MemberService.scala

```diff
@@ -456,21 +458,20 @@ class MemberService(identityService: IdentityService,
     val contribute = zuoraService.getFeatures.map { features =>
       val planId = planChoice.productRatePlanId
       val ratePlanChargeId = catalog.contributor.charges.chargeId.get
-
       val plan = RatePlan(productRatePlanId = planId.get,
                           chargeOverride = Some(ChargeOverride(productRatePlanChargeId = ratePlanChargeId,
                                                 price = Some(joinData.payment.amount))))
       val currency = GBP
       val today = DateTime.now.toLocalDate
-
-      Contribute(account = createAccount(contactId, identityId, currency, joinData.payment),
+      val transactingCountry = Country.UK // TODO get from GEO-IP
```

Comment:
This method may be deletable if there are no and not to be any Recurring Contributions going through membership-frontend.